owncloud/core
# Remove obsolete upload size section

## Description
Removes max upload size settings section.

The admin.js file had additional code related to "sharingaim", I checked back to stable8 and this element doesn't exist anywhere, so these might be relics from ancient deleted features.

## Related Issue
Fixes https://github.com/owncloud/core/issues/27672

## Motivation and Context
The web UI now uses Webdav PUT instead of HTTP POST, so there is no upload size limit any more.
We remove the settings section to avoid confusion.

## How Has This Been Tested?
- verify that the section has disappeared
- click through other settings sections and check that there are no warnings in owncloud.log about missing stuff

## Screenshots (if appropriate):

## Types of changes


## Checklist:



@tomneedham please review
